docs/reference/tab-widgets/troubleshooting/watermark/troubleshooting-watermark.asciidoc

```diff
@@ -0,0 +1,25 @@
+// tag::cloud[]
+
+* {ref}/docs-delete.html[Delete unused data].
+* {cloud}/ec-customize-deployment-components.html[Increase the disk size on your Hot data and Content tier (scale up)]
+
+NOTE: If your {es} cluster is unhealthy and reports a status of red, then increasing the disk size of your Hot data and Content tier may fail. You might need to delete some data so the configuration can be edited. If you want to increase your disk size without deleting data, then {cloud}/ec-get-help.html[reach out to Elastic support] and we will assist you with scaling up.
+
+Also, for future prevention, you can do below:
+
+* {cloud}/ec-customize-deployment-components.html[Increase the disk size on your Hot data and Content tier (scale up)]
```

Comment:
It's invalid to restrict this to only hot/content tier, it can happen on any data tier